source/includes/management-api/_management-api.md

```diff
@@ -8125,14 +12776,13 @@ managementAPIToken ( Scopes: read:companies )
 ### Count Events
 
 <aside class="info">
-The Moesif UI has a query generator to make it easy to create the necessary query. From any workspace view in Moesif, click teh orange Embed  button and then select Access via API.
```

Comment:
Looks like these were manually added. Can we add all the callounts for "The Moesif UI has a query generator to make..." back in right spots?